### ISSUE TITLE
app crashed when setupMarkerCluster

### ISSUE THREAD

**Issue by weijyewang** [naderio/nativescript-google-maps-utils#13]
I'm trying to enable clustered marker in android using angular 6.

My code was rather simple, and I had no idea when the app crashed (it exited to the emulator home screen automatically) when it ran this line of code
`GoogleMapsUtils.setupMarkerCluster(this.mapView, markerSet);`

Here's the code snippet:
import section:

```
import { MapView, Marker, Position } from 'nativescript-google-maps-sdk';
import * as GoogleMapsUtils from "nativescript-google-maps-utils"

```
inside the function
```
let markerSet = [];
this.journeys.forEach(journey => {
	let theMarker: Marker = new Marker();
	theMarker.position = Position.positionFromLatLng(journey.latitude, journey.longitude);
	theMarker.title = journey.vehicleRegistrationNumber;
        theMarker.snippet = journey.vehicleRegistrationNumber;
	markerSet.push(theMarker);
});

setTimeout(() => {
	GoogleMapsUtils.setupMarkerCluster(this.mapView, markerSet);
	this.zoomCenterOnMultipleMarkers();
}, 1000)
```

Does anyone get it to working on angular? Please help

**Comment by sedrak1987** [comment]
Does anyone get it to working on angular?

**Comment by fawadmukhtar** [comment]
remove this `mapView._markers.push(marker);` statement form this method `setupMarkerCluster()` in `index.android.ts`.

There is no `_markers` property in the MapView class

Secondly, There should be minimum 5 markers to showing cluster

**Comment by sedrak1987** [comment]
dear where is index.android.ts file?(maybe in node_modules, if yes, then why I must change core files)

**Comment by fawadmukhtar** [comment]
@sedrak1987  because the app is crashing at that point. you need to fork it and update that file to get a expected result

**Comment by sedrak1987** [comment]
yes, but after plugin update or node_modules remove in every time I need remove that line. Right?

**Comment by fawadmukhtar** [comment]
No, when you fork it and use your own. that's control from your package.json file

**Comment by sedrak1987** [comment]
can you give me instruction steps how do it, please?

**Comment by weijyewang** [comment]
@sedrak1987  
1. Fork this project in Github
![image](https://user-images.githubusercontent.com/8890603/47073704-726d6a00-d22b-11e8-98ae-ee313293a6ba.png)
2.  Do whatever changes neccessary like what @fawadmukhtar  suggested in your forked new repo.
3. Publish to npm or Nativescript  plugin market (https://docs.nativescript.org/plugins/building-plugins#step-4-publish-your-plugin)
4. Install back your published version.

**Comment by sedrak1987** [comment]
Thanks a lot. Will try later when will get free time

**Comment by weijyewang** [comment]
Remember to uninstall this original nativescript plugin before using your forked version. Else you will have namespace conflict

**Comment by Zampa98** [comment]
Can anyone help me? I can not publish my fork
